plugins/modules/zypper.py

```diff
@@ -586,7 +632,7 @@ def main():
         elif state in ['installed', 'present', 'latest']:
             packages_changed, retvals = package_present(module, name, state == 'latest')
 
-    retvals['changed'] = retvals['rc'] in [0, 102] and bool(packages_changed)
+        retvals['changed'] = retvals['rc'] in [0, 102] and bool(packages_changed)
```

Comment:
This was an unintentional change.  Not clear why it is showing indented by may have been caused by my attempt at merge conflict resolution.  I will revert this change.